aws.tf

```diff
@@ -15,10 +15,10 @@ terraform {
     key            = "ae-apps.tfstate"
     region         = "us-east-1"
     dynamodb_table = "terraform-lock-table"
+    profile        = "aeternity"
```

Comment:
Please don't hardcode local files/names/profiles etc. everyone should be free to use different profile name. Even more depending on setup/auth you might need more than one.